### ISSUE TITLE
PartDesign  → NewSketch

### ISSUE THREAD

**Issue by kaktusus** [FreeCAD/FreeCAD-translations#155]
```
OS: Debian GNU/Linux bookworm/sid (KDE/plasma)
Word size of FreeCAD: 64-bit
Version: 0.20.28909 (Git) AppImage
Build type: Release
Branch: (HEAD detached at 2084652)
Hash: 20846523c9d8219ec0fe33c499a64e700d2266b6
Python 3.9.12, Qt 5.12.9, Coin 4.0.0, OCC 7.5.3
Locale: Polish/Poland (pl_PL)
Installed mods: 
...
```
Labels do not have translations in the GUI.

![image](https://user-images.githubusercontent.com/39754351/169662519-d731df32-8c4f-470d-be83-b007dd6ec60f.png)

1. `XY_Plane` - no string in Crowdin.
    https://crowdin.com/translate/freecad/all/en-pl?filter=basic&value=0#q=XY_Plane
2. `XZ_Plane` - no string in Crowdin.
    https://crowdin.com/translate/freecad/all/en-pl?filter=basic&value=0#q=XZ_Plane
3. `YZ_Plane` - no string in Crowdin.
    https://crowdin.com/translate/freecad/all/en-pl?filter=basic&value=0#q=YZ_Plane

In older versions of the program the comment in brackets was more intuitive - it specified the position of the plane view in space 
e.g.
base plane top view
base plane front view
base plane side view
or something similar - I don't remember the details.
It would be good to go back to that.

**Comment by chennes** [comment]
@PaddleStroke I don't know if this is on your radar, but unifying this bit of UI would be a nice project 😄 .

**Comment by PaddleStroke** [comment]
Yes this is on my mind for some times. It seems there are so many different placement UI. It needs to be unified.

**Comment by kaktusus** [comment]
still needs attention.

**Comment by hasecilu** [comment]
On 1.0 is translated

![Image](https://github.com/user-attachments/assets/eb26a8df-fb40-4f89-983f-70e73a5bac09)

but on 1.1 not but it seems because on FreeCAD/FreeCAD@fe60505c17 context changed from     `<name>App::Origin</name>` to `<name>App::OriginGroupExtension</name>`. Translation were already done so only an update seems required. So maybe close this.

![Image](https://github.com/user-attachments/assets/e6da47a0-9007-474b-9fc8-08339449813a)

**Comment by kaktusus** [comment]
I only close tickets when I see a working translation in the GUI.
The problem is when I forget how to reproduce the bug. 😄 


The string `base plane` could be individually adjusted for each occurrence, e.g., on the side, at the back, from above, etc.
Then it will be more communicative. @[sifabiancic](https://github.com/sifabiancic) what do you think about this?

**Comment by kaktusus** [comment]
```
OS: Debian GNU/Linux 13 (trixie) (KDE/plasma/wayland)
Architecture: x86_64
Version: 1.1.0dev.14555 (Git shallow) AppImage
Build date: 2025/10/14 15:14:51
Build type: Release
Branch: main
Hash: 41ce37233d64c81c2f105de085635d4dcd669943
Python 3.11.14, Qt 6.8.3, Coin 4.0.3, Vtk 9.3.1, boost 1_86, Eigen3 3.4.0, PySide 6.8.3
shiboken 6.8.3, xerces-c 3.3.0, IfcOpenShell 0.8.2, OCC 7.8.1
Locale: Polish/Poland (pl_PL)
Navigation Style/Orbit Style/Rotation Mode: Touchpad/Rounded Arcball/Window center
Stylesheet/Theme/QtStyle: FreeCAD Dark.qss/FreeCAD Dark/
Logical DPI/Physical DPI/Pixel Ratio: 81/141.951/1
Installed mods: 
  * Telemetry 1.0.2
  * sheetmetal 0.7.24
  * Reinforcement
```


<img width="372" height="664" alt="Image" src="https://github.com/user-attachments/assets/6fff1504-0967-421b-be86-c52da59b3634" />
